docs/controls/WebView.md

```diff
@@ -0,0 +1,314 @@
+{
+    if (args.PermissionRequest.PermissionType == WebViewPermissionType.Geolocation &&
+        args.PermissionRequest.Uri.Host == "www.bing.com")
+    {
+        args.PermissionRequest.Allow();
+    }
+}
+```
+
+If your app requires user input or other asynchronous operations to respond to a permission request, use the [Defer](https://docs.microsoft.com/uwp/api/windows.web.ui.webviewcontrolpermissionrequest.defer) method of [WebViewControlPermissionRequest](https://docs.microsoft.com/uwp/api/windows.web.ui.webviewcontrolpermissionrequest) to create a [WebViewControlDeferredPermissionRequest](https://docs.microsoft.com/uwp/api/windows.web.ui.webviewdeferredpermissionrequest) that can be acted upon at a later time.
+
+## Interact with web view content
+
+You can interact with the content of the web view by using the [InvokeScriptAsync](https://docs.microsoft.com/uwp/api/windows.web.ui.interop.webviewcontrol.invokescriptasync) method to invoke or inject script into the web view content, and the [ScriptNotify](https://docs.microsoft.com/uwp/api/windows.web.ui.interop.webviewcontrol.scriptnotify) event to get information back from the web view content.
+
+To invoke JavaScript inside the web view content, use the [InvokeScriptAsync](https://docs.microsoft.com/uwp/api/windows.web.ui.interop.webviewcontrol.invokescriptasync) method. The invoked script can return only string values.
+
+For example, if the content of a web view named `webView1` contains a function named `setDate` that takes 3 parameters, you can invoke it like this.
+```csharp
+string[] args = {"January", "1", "2000"};
+string returnValue = await webView1.InvokeScriptAsync("setDate", args);
+```
+
+You can use **InvokeScriptAsync** with the JavaScript **eval** function to inject content into the web page.
+
+Here, the text of a XAML text box (`nameTextBox.Text`) is written to a div in an HTML page hosted in `webView1`.
+
+```csharp
+private async void Button_Click(object sender, RoutedEventArgs e)
+{
+    string functionString = String.Format("document.getElementById('nameDiv').innerText = 'Hello, {0}';", nameTextBox.Text);
+    await webView1.InvokeScriptAsync("eval", new string[] { functionString });
+}
+```
+
+Scripts in the web view content can use **window.external.notify** with a string parameter to send information back to your app. To receive these messages, handle the [ScriptNotify](https://docs.microsoft.com/uwp/api/windows.web.ui.interop.webviewcontrol.scriptnotify) event.
+
+## Options for web content hosting
+
+You can use the [Settings](https://docs.microsoft.com/uwp/api/windows.web.ui.interop.webviewcontrol.settings) property (of type [WebViewControlSettings](https://docs.microsoft.com/uwp/api/windows.web.ui.interop.webviewcontrolsettings) to control whether JavaScript and IndexedDB are enabled. For example, if you use a web view to display strictly static content, you might want to disable JavaScript for best performance.
+
+ 
+
```

Comment:
Make sure to add the Requirements and Source Code section of the [docs .template](
Requirements)

Reply:
Added in 0ab5ecc4e0bd35a6263c11641153c61d6ce45b16
CC @mcleanbyron @normesta

Reply:
@nmetulev @mcleanbyron @normesta Doc updates are in #8.